packages/install-expo-modules/package.json

```diff
@@ -16,7 +16,7 @@
     "build"
   ],
   "scripts": {
-    "build": "ncc build ./src/index.ts -o build/",
+    "build": "ncc build --external simple-plist ./src/index.ts -o build/",
```

Comment:
in `packages/install-expo-modules/build/index.js` most requires have this form: `__nccwpck_require__(92740)`. But `simple-plist` keeps referring to relative paths: `require("./parse")`. And that fails.

Reply:
I'm slightly concerned by this, as `build:prod` should bundle everything. If somehow `simple-plist` doesn't bundle in `build`, I think it won't bundle in `build:prod` too.